tests/Makefile

```diff
@@ -8,23 +8,24 @@ TOPLEVEL=$(CURDIR)/..
 .PHONY: all
 all: test
 	@echo "+ $@"
+	@GOTAGS=$(GOTAGS),test,test_e2e ../scripts/go-test.sh -cover -v -run Test 2>&1 | tee test.log
 	@$(MAKE) report JUNIT_OUT=all-tests-results
```

Comment:
This calls the `test` target inside `stackrox.mk`. I think there are two options here:

1. Remove the call to the `test` target since you are already executing the tests in line 11.
2. Add the `test_e2e` tag in the `test` target.

If we go with (1), I think you should add the previously used test flags to line 11 (`TESTFLAGS=-race -p 1 -timeout 30m`).

If we go with (2), we need to make sure that no other tests are using the `test` target because otherwise they might stop working.

[nit] I don't think `-run Test` is necessary here.

Reply:
I've checked and stackrox.mk is not included anywhere else but this Makefile. That should make option 2 the safer and better choice.

The good news is that in the [last CI run](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/stackrox_stackrox/12232/pull-ci-stackrox-stackrox-master-gke-nongroovy-e2e-tests/1818681671751307264) all tests were already being executed so there's that. I'll run upgrade and compliance tests, feel free to merge this if they pass.